app/controllers/admins/ws_panel_controller.rb

```diff
@@ -226,4 +227,14 @@ def edit_driver
     end
   end
 
+  def get_dr_avatar
```

Comment:
Please give more informative names to methods. For example, `driver_avatar`.
[Also, you shouldn't name actions with prefix `get`](https://github.com/bbatsov/ruby-style-guide#accessor_mutator_method_names).